src/HelloWorldPubSub/HelloWorldPub.c

```diff
@@ -61,16 +61,17 @@ main ( int argc, char *argv[] )
     solClient_session_createFuncInfo_t sessionFuncInfo = SOLCLIENT_SESSION_CREATEFUNC_INITIALIZER;
 
     /* Session Properties */
-    const char     *sessionProps[20];
+    const char     *sessionProps[50];
```

Comment:
This change seems unneeded?

Reply:
The addition of the password parameter necessitates more space. This is overcompensating, but there are segfault issues without the increase.

Reply:
Reasonable then.